substrate/client/statement-store/src/lib.rs

```diff
@@ -531,26 +562,26 @@ impl Store {
 		Client::Api: ValidateStatement<Block>,
 	{
 		let mut path: std::path::PathBuf = path.into();
-		path.push("statements");
+		path.push("statements2");
```

Comment:
Can we not just purge the statements stores instead of creating a second folder?